fieldservice_account/tests/test_fsm_account.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo.tests.common import TransactionCase
+
+
+class FSMAccountCase(TransactionCase):
+    def setUp(self):
+        super(FSMAccountCase, self).setUp()
+        self.Wizard = self.env["fsm.wizard"]
+        self.WorkOrder = self.env["fsm.order"]
+        self.AccountInvoice = self.env["account.move"]
+        self.AccountInvoiceLine = self.env["account.move.line"]
+        # create a Res Partner
+        self.test_partner = self.env["res.partner"].create(
+            {"name": "Test Partner", "phone": "123", "email": "tp@email.com"}
+        )
+        # create a Res Partner to be converted to FSM Location/Person
+        self.test_loc_partner = self.env["res.partner"].create(
+            {"name": "Test Loc Partner", "phone": "ABC", "email": "tlp@email.com"}
+        )
+        self.test_loc_partner2 = self.env["res.partner"].create(
+            {"name": "Test Loc Partner 2", "phone": "123", "email": "tlp@example.com"}
+        )
+        # create expected FSM Location to compare to converted FSM Location
+        self.test_location = self.env["fsm.location"].create(
+            {
+                "name": "Test Location",
+                "phone": "123",
+                "email": "tp@email.com",
+                "partner_id": self.test_loc_partner.id,
+                "owner_id": self.test_loc_partner.id,
+                "customer_id": self.test_loc_partner.id,
```

Comment:
```suggestion
```
customer_id was moved to fieldservice_account_analytic

Reply:
Good catch @RLeeOSI

I'll take care of that.

Do you have any thoughts about my questions here: [https://github.com/OCA/field-service/pull/598#issuecomment-661327008](https://github.com/OCA/field-service/pull/598#issuecomment-661327008)?

Reply:
Yes, a migration script is needed. However, the v13 account migration is not done yet (see the latest [here](https://github.com/OCA/OpenUpgrade/issues/1733)) so it might be best to put that on hold for now. According to the (maintainer guide)[https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration] all old migration scripts should be removed